### PR TITLE
being able to force/ask user to update app #226

### PR DESCRIPTION
**Pull request by jknnr**
https://github.com/teamforus/me/issues/226

### CODE REVIEW

**Comment by GerbenBosschieter** [comment]
@jknnr @maestrovsmaster @maxvisser Can someone explain me how to test this?

Wat I did:
- Removed production app (downloaded from store)
- Build a signed APK from the force update user branch with a lower version than compared to production.
- Installed the APK

Result: nothing.

From what I understand:
The backend reads if the version of the app is lower than there is available. If true give notification?

**Comment by jknnr** [comment]
@GerbenBosschieter try once again

File: presentation/src/main/res/layout/view_snackbar_update_app.xml [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+                android:layout_height="wrap_content"
+                android:layout_centerVertical="true"
+                android:layout_marginStart="8dp"
+                android:layout_marginEnd="8dp"
+                android:layout_toStartOf="@id/update"
+                android:layout_toEndOf="@id/image"
+                android:gravity="start"
+                android:lines="2"
+                android:padding="8dp"
+                android:text="New update is available,\n update your application!"
```
Comment (maxvisser) [inline presentation/src/main/res/layout/view_snackbar_update_app.xml:45]: can this also be added to strings?

Dutch text is: 
Er is een nieuwe update beschikbaar!

Comment (maestrovsmaster) [reply presentation/src/main/res/layout/view_snackbar_update_app.xml:45]: ok!

**Pull request merged**